deps/Makefile

```diff
@@ -161,6 +161,15 @@ libconfig/libconfig/lib/.libs/libconfig++.a:
 
 libconfig: libconfig/libconfig/lib/.libs/libconfig++.a
 
+prometheus-cpp/lib/libprometheus-cpp-core.a:
+	cd prometheus-cpp && tar --strip-components=1 -zxf v0.9.0.tar.gz -C .
```

Comment:
Please do not use this form of unpacking the tarball.
Please follow the same convention for any other dependencies: every version has its own directory, and we use a symlink.

Reply:
Done! Everything follows the convention now.